### PR TITLE
Fix large job example

### PR DESCRIPTION
**Pull request by aelmanaa**
Fix Connect any API [large job example](https://docs.chain.link/docs/large-responses/)

### CODE REVIEW

**Review by dwightjl — CHANGES_REQUESTED** [review]

File: docs/Using Any API/large-responses.md [hunk 1]
```diff
@@ -43,8 +47,6 @@ To consume an API with a large responses, your contract should inherit from [Cha
 
 The job spec for the Chainlink node in this example can be [found here](../example-job-spec-large/).
```
[the remark this answers is not in the document]
Comment (dwightjl) [reply docs/Using Any API/large-responses.md:48]: Ignore. I see we have the update in https://github.com/smartcontractkit/documentation/pull/673/files

File: docs/Using Any API/large-responses.md [hunk 2]
```diff
@@ -43,8 +47,6 @@ To consume an API with a large responses, your contract should inherit from [Cha
...
 
-If the LINK address for targeted blockchain is not [publicly available](../link-token-contracts/) yet, replace [setPublicChainlinkToken(/)](../chainlink-framework/#setpublicchainlinktoken) with [setChainlinkToken(_address)](../chainlink-framework/#setchainlinktoken) in the constructor, where `_address` is a corresponding LINK token contract.
-
 ## Choosing an Oracle and JobId
 
 `oracle` refers to a specific Chainlink node that a contract makes an API call from, and `specId` refers to a specific job for that node to run. Each job is unique and returns different types of data.
```
[the remark this answers is not in the document]
Comment (aelmanaa) [reply docs/Using Any API/large-responses.md:52]: good catch! solved

File: docs/Using Any API/large-responses.md [hunk 3]
```diff
@@ -13,24 +18,24 @@ This page explains how to make an HTTP GET request to an external API from a sma
...
 - Callback function signature
 
->❗️ Note on Funding Contracts
+> ❗️ Note on Funding Contracts
 >
 > Making a GET request will fail unless your deployed contract has enough LINK to pay for it. **Learn how to [Acquire testnet LINK](../acquire-link/) and [Fund your contract](../fund-your-contract/)**.
->
 
 ```solidity Kovan
 {% include 'samples/APIRequests/GenericBigWord.sol' %}
```
Comment (dwightjl) [inline docs/Using Any API/large-responses.md:41]: Add instructions for how to test this example.

File: docs/Using Any API/large-responses.md [hunk 4]
```diff
@@ -41,15 +46,13 @@ To consume an API with a large responses, your contract should inherit from [Cha
     <a href="/docs/conceptual-overview/#what-is-remix" >What is Remix?</a>
 </div>
 
-The job spec for the Chainlink node in this example can be [found here](../example-job-spec-large/).
-
-If the LINK address for targeted blockchain is not [publicly available](../link-token-contracts/) yet, replace [setPublicChainlinkToken(/)](../chainlink-framework/#setpublicchainlinktoken) with [setChainlinkToken(_address)](../chainlink-framework/#setchainlinktoken) in the constructor, where `_address` is a corresponding LINK token contract.
+Note that this example contract is hardcoded to work on _Kovan_ testnet. The job spec for the Chainlink node in this example can be [found here](../example-job-spec-large/).
 
-## Choosing an Oracle and JobId
+## Choosing Link token address, Oracle and JobId
```
Comment (dwightjl) [inline docs/Using Any API/large-responses.md:51]:
```suggestion
## Setting the LINK token address, Oracle, and JobId
```

**Review by dwightjl — CHANGES_REQUESTED** [review]

**Pull request closed without merging**